src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -11769,41 +11769,12 @@ def test_aks_azure_service_mesh_with_egress_gateway(
         # enable egress gateway
         update_cmd = (
             "aks mesh enable-egress-gateway --resource-group={resource_group} --name={name} "
-            "--egress-gateway-nodeselector istio=egress"
         )
         try:
             self.cmd(
                 update_cmd,
                 checks=[
                     self.check("serviceMeshProfile.mode", "Istio"),
-                    self.check(
-                        "serviceMeshProfile.istio.components.egressGateways[0].nodeSelector.istio",
-                        "egress",
-                    ),
-                    self.check(
```

Comment:
We don't need to remove the basic enabled check right?

Reply:
As discussed offline, this does the basic enabled check on line 11778